install_cexts.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+import os
+import subprocess
+import argparse
+import imp
+
+
+DIST_CEXT = 'kolibri/dist/cext'
+PYPI_DOWNLOAD = 'https://pypi.python.org/simple/{package_name}'
+
+
+"""
+Check if requests and bs4 modules are installed.
+"""
+def _install_requests_and_bs4_if_needed():
+    try:
+        imp.find_module('requests')
+        print ('requests module exists')
+    except:
```

Comment:
I'm nitpicking here, but having `except` catching ALL exceptions is an anti-pattern always - it's better to write `except ImportError` or whatever specific exception you want to catch. Small thing here, but good habit!